flask_session_cookie_manager.py

```diff
@@ -0,0 +1,108 @@
+""" Flask Session Cookie Decoder/Encoder """
+__author__ = 'Wilson Sumanang, Alexandre ZANNI'
+
```

Comment:
> You converted the space indentation to TABs indentation. Please go back to 4 spaces.
Fixed



> I assume either you removed `session_cookie_manager.py` and uploaded `flask_session_cookie_manager.py` via Github webui or you used `mv` to rename the file and your text editor/IDE is aggressively converting space to TABs.

Not sure why did this happen, because I 100% used git tool to upload changes. Thanks for the "git mv" top though :)